tasks/MatchedCatalogsAnalysis.py

```diff
@@ -43,3 +42,36 @@ def __init__(self, config, *args, **kwargs):
 
     def run(self, matchedCatalog):
         return self.measure.run(matchedCatalog, self.config.connections.metric)
+
+
+class MatchedMultiCatalogAnalysisTaskConnections(MetricConnections,
+                                                 dimensions=("tract", "patch", "abstract_filter",
+                                                 "instrument", "skymap")):
+    matchedCatalogMulti = pipeBase.connectionTypes.Input(doc="Input matched catalog.",
+                                                    dimensions=("tract", "patch", "instrument"),
+                                                    storageClass="SimpleCatalog",
+                                                    name="matchedCatalogMulti")
+    measurement = pipeBase.connectionTypes.Output(doc="Resulting matched catalog.",
+                                                  dimensions=("tract", "patch",
+                                                              "instrument","abstract_filter"),
+                                                  storageClass="MetricValue",
+                                                  name="metricvalue_{package}_{metric}")
+
+
+class MatchedMultiCatalogAnalysisTaskConfig(MetricConfig,
+                               pipelineConnections=MatchedMultiCatalogAnalysisTaskConnections):
```

Comment:
Again, this is identical to `MatchedCatalogAnalysisTaskConfig` I think.

Reply:
See prev. comment - it needs the Multi connections class to match over all filters.